conans/client/conanfile/configure.py

```diff
@@ -38,3 +43,11 @@ def run_configure_method(conanfile, down_options, profile_options, ref):
     if hasattr(conanfile, "build_requirements"):
         with conanfile_exception_formatter(conanfile, "build_requirements"):
             conanfile.build_requirements()
+
+
+def _conanfile_config_options_default(conanfile):
+    handle_common_config_options(conanfile)
+
+
+def _conanfile_configure_default(conanfile):
+    handle_common_configure_options(conanfile)
```

Comment:
Still boilerplate? What is the purpose of these functions? I'd remove them and call the ``handle_xxxx`` directly in the above calls.

Reply:
ok